src/components/DebtListItem.jsx

```diff
@@ -5,15 +5,34 @@ class DebtList extends Component {
   render() {
     const debt = this.props.debt;
 
-    return (
-      <li className="DebtListItem">
-        <h2>{debt.description}</h2>
-        <dl>
-          <dt>Total Principle</dt><dd>{Currency(debt.principle)}</dd>
-          <dt>Lifetime</dt><dd>{debt.lifetime} months</dd>
-        </dl>
-      </li>
-    );
+    let rtn = null;
```

Comment:
- [ ] instead of storing the return value in `rtn`, just return the values form the switch statement (you won't need this variable then)